neo/Consensus/ConsensusService.cs

```diff
@@ -428,6 +433,14 @@ private void OnPrepareRequestReceived(ConsensusPayload payload, PrepareRequest m
                 if (context.CommitPayloads[i]?.ConsensusMessage.ViewNumber == context.ViewNumber)
                     if (!Crypto.Default.VerifySignature(hashData, context.CommitPayloads[i].GetDeserializedMessage<Commit>().Signature, context.Validators[i].EncodePoint(false)))
                         context.CommitPayloads[i] = null;
+
+            if (context.TransactionHashes.Length == 0)
```

Comment:
@shargon, is this also because of the removal of minner transaction?

Reply:
Yes, It's possible, but without transactions, there are no blocks :S